nuxeo-web-ui-ftest/webdriver/test/features/step_definitions/support/fixtures/users.js

```diff
@@ -37,5 +37,13 @@ fixtures.users = {
 
 module.exports = function () {
   this.After(() => Promise.all(Object.keys(users)
-      .map((user) => user !== 'Administrator' ? fixtures.users.delete(user) : Promise.resolve())));
+      .map((user) => {
+        if (user !== 'Administrator') {
+          const userWorkspace = `/default-domain/UserWorkspaces/${user}`;
+          nuxeo.repository().delete(userWorkspace).catch(() => {});
+          fixtures.users.delete(user);
```

Comment:
Because user workspaces are returned as search results. So if it is created during the tests but not removed, the number of test results for certain searches will be inconsistent depending on the order in which the tests are run.